src/components/Navbar/DonateBtn.js

```diff
@@ -27,6 +27,7 @@ const StyledFont = styled.a`
   display: inline-block;
   padding: 0px;
   padding-right: 8px;
+  font-weight: 500
```

Comment:
files not part of "burger menu" should have their separate pull request ideally

Reply:
@harshvchawla  I think for a such one line minor modification we don't need a separate PR as navebar and sidebar are in the same section and part of my assignments.

Reply:
@BBoutet1  you are right; I was just thinking this was part of Donate button but makes sense to fix such minor stuff when we get a chance